src/sections/hero.html

```diff
@@ -0,0 +1,80 @@
+                </g>
+                <defs>
+                  <clipPath id="clip0_405_90">
+                    <rect width="107" height="107" fill="white"/>
+                  </clipPath>
+                </defs>
+              </svg></span>
```

Comment:
Make sure to create an external file for this SVG path! and apply it in the parts of your codebase where it's truly necessary!